packages/common/src/FormattingUtils.js

```diff
@@ -1,4 +1,4 @@
-const networkUtils = require("./PublicNetworks");
+const { PublicNetworks: networkUtils } = require("./PublicNetworks");
```

Comment:
Importing like this has been a bug for a while which has caused `createEtherscanLinkFromtx` to always return `etherscan.io` by default instead of the specific network block explorer url.